package_control/downloaders/wget_downloader.py

```diff
@@ -235,8 +234,8 @@ def parse_output(self, clean_run):
             HTTP header names.
         """
 
-        with open_compat(self.tmp_file, 'r') as f:
-            output = read_compat(f).splitlines()
+        with open(self.tmp_file, 'r', encoding='utf-8', errors='replace') as fobj:
+            output = fobj.read().splitlines()
```

Comment:
`fobj.readlines()`? Not sure off-hand whether that keeps line endings.

Reply:
Yes, it does.